.github/workflows/ci.yml

```diff
@@ -33,10 +33,10 @@ jobs:
           cache: pnpm
 
       - name: Install dependencies
-        run: pnpm install --ignore-scripts
+        run: pnpm install --ignore-scripts --frozen-lockfile
```

Comment:
Explicitly request `--frozen-lockfile`, as the [implicit default](https://pnpm.io/cli/install#--frozen-lockfile) turned out to be unreliable.